### DISCUSSION TITLE
Game Selection Menu

### DISCUSSION THREAD

**Discussion by elroykanye** [osscameroon/tg-gamebot#27] [Ideas]
Hello @andrew21-mch , I think we should communicate and resolve the various the ideas and implementations we will make for that selection menu. 

I propose we start first with the initial design you had. How do you see?

**Comment by elroykanye** [comment]
In the case sub tasks come from this discussion, we will directly create issues that should address them from here :)

**Reply by andrew21-mch** [reply]
Ok 
Should we do that now?

**Reply by elroykanye** [reply]
I think it is already late. How about tomorrow when we meet at Nasia

**Reply by elroykanye** [reply]
Then finish up in the evening.

**Reply by andrew21-mch** [reply]
Hi @elroykanye good evening, how you doing, could we do this now?

**Reply by elroykanye** [reply]
Absolutely

**Reply by andrew21-mch** [reply]
> Absolutely

so where do we start

**Comment by andrew21-mch** [comment]
so where do we start

**Comment by elroykanye** [comment]
Hello Drew, let's split this issue into 2. We have actually built the command selection menu and not the help selection menu

**Reply by andrew21-mch** [reply]
so what do you think, we started with the help and i guess the outlay is there already
what was needed was a cool description for the help section which we were still to be out

**Reply by elroykanye** [reply]
Okay then,
- I will create an issue for the commands menu, then update the issue that this pr fixes.
- You will finish with that help callback, and I will add any necessary stuff that we need to add.
- You will then mark it as ready to resolve so we can review.

**Reply by andrew21-mch** [reply]
Ok then
